src/providers/ContentShareProvider/docs/ContentShareProvider.mdx

```diff
@@ -22,8 +36,17 @@ The `ContentShareProvider` provides state and functionality for content sharing.
   // Whether or not the local user's content share is loading
   isLocalShareLoading: boolean;
 
-  // The chime attendee ID of the user sharing
-  sharingAttendeeId: string | null;
+  // Whether the user can start a content share based on current limits and `isLocalUserSharing`
```

Comment:
We can expand a bit more on "current limits" for clarity here. Is this just the `maxContentShares`?

Reply:
Yeah, let me update it, it should be `maxContentShares`, current limits is not very clear.

Reply:
Seems like still needs an update?